File: Day-09/Pattern9.js
function PatternNine(n) {
  let i = 1;
  while (i <= n) {
    let j = 1;
    let count = i;
    let str = "";
    while (j <= i) {
      str += count;
      count++;
      j++;
    }
    i++;
    console.log(str);
  }
}

PatternNine(4);

File: Day-09/Pattern9.py
n = int(input())

def PatternNine(n):
    i = 1
    while(i <=n):
        count = i
        j = 1
        while(j<=i):
            print(count, end='')
            count = count +1
            j = j + 1
        i = i+ 1
        print(end='\n')

PatternNine(n)


def PatternNineNoCount(n):
    i = 1
    while(i <=n):
        j = 0
        while(j<i):
            print(i+j,  end='')
            j = j + 1
        i = i+ 1
        print(end='\n')

PatternNineNoCount(n)